docs/ALIASES.md

```diff
@@ -97,13 +97,27 @@ Here's an example of a bash script which returns the the flag key as it's own al
 ```yaml
 aliases:
   - type: command
-    command: ./launchdarkly/launchdarklyAlias.sh # must be a valid shell command.
+    command: ./.launchdarkly/flagAlias.sh # must be a valid shell command.
     timeout: 5 # seconds
 ```
 
-Contents of `./launchdarkly/launchdarklyAlias.sh`:
+Contents of `./.launchdarkly/flagAlias.sh`:
```

Comment:
this is unrelated, but we have people add config.yaml to `.launchdarkly` directory, so i thought using that as an example makes more sense.